### DISCUSSION TITLE
HMS/HMT compatibility

### DISCUSSION THREAD

**Discussion by PetrolDriver** [tbnobody/OpenDTU#336] [Ideas]
Hi,

the HMS and HMT series also uses 2.4 GHz wireless connection for communicating with the DTU.
Do you think that OpenDTU also will support the HMS or HMT series?

Thanks.

**Comment by FrodoVDR** [comment]
Unfortunately, HMS and HMT uses Sub-1g, which uses other frequencies, probably 868 MHz. Since this requires a completely different transceiver and possibly also the protocol is different, there is unfortunately no support from OpenDTU yet. 

I would also be happy if HMS and HMT were supported, as I operate two HMS-1800s myself.

**Reply by PetrolDriver** [reply]
Thanks for answer.

**Reply by stefan123t** [reply]
Maybe check our issue / feature request in AhoyDTU for more details and hardware / traces needed ?

https://github.com/lumapu/ahoy/issues/233

**Comment by PetrolDriver** [comment]
Thanks

**Reply by PetrolDriver** [reply]
HMS-500 works in Europe and United States according to the manufacturer's information via 915 MHz (not on 868 MHz).

**Comment by FrodoVDR** [comment]
Gibt es hierzu was neues zu berichten? 

Es gibt seit ein paar Wochen in Discord eine OpenDTU Testversion, allerdings findet sich hier in Github keinerlei Hinweise auf die Integration.

Auch in der Roadmap https://github.com/users/tbnobody/projects/4 finde ich hierzu nichts...

**Comment by tbnobody** [comment]
It's done when its done...

Aber es wird nicht mehr so lange dauern wie es schon gedauert hat. Konnte über die Osterfeiertage einige Baustellen schließen

**Comment by alekob** [comment]
Is it or will it be possible to use a "E49-900T20D" board instead of a "E49-900M20S" (both EBYTE)? The first one would be easier to handle in a DIY project

**Reply by tbnobody** [reply]
The E49-900T20D has a UART interface instead of a SPI interface. Therefore, Not supported
